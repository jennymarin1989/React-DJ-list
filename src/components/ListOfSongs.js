import React from 'react';

const ListOfSongs = (props) => {

    return(

        <div>
           <h1>List of songs</h1>
           <h2>{props.data}</h2>
        </div>    
    )

}

export default ListOfSongs