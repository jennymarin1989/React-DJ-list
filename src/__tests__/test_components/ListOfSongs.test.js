import React from 'react';
import { shallow } from 'enzyme';
import ListOfSongs from '../../components/ListOfSongs'


describe("listOfSongs", () => {

    let list = shallow(<ListOfSongs />)
  
  it('renders correctly', ()=>{
    expect(list).toMatchSnapshot();
  })
})