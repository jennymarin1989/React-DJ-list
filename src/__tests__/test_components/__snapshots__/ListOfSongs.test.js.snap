// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`listOfSongs renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ListOfSongs />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          List of songs
</h1>,
        <h2 />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "List of songs",
        },
        "ref": null,
        "rendered": "List of songs",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            List of songs
</h1>,
          <h2 />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "List of songs",
          },
          "ref": null,
          "rendered": "List of songs",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
